client/localization/elements/viewoption_en.json

```diff
@@ -35,7 +35,7 @@
   "viewoption:sideBySide": "Side by side",
   "viewoption:sidenotes": "Show as sidenotes",
   "viewoption:textView": "Original text and translation",
-  "viewoption:textViewDescription": "Some texts on SuttaCentral are “segmented”; the original text and translation are approximately matched line by line. For such texts, see both root text and translation.",
+  "viewoption:textViewDescription": "Some texts on SuttaCentral are “segmented”; the original text and translation are approximately matched line by line. For such texts, see both root text and translation. Press <code class=\"key\">V</code> to toggle between the views.",
```

Comment:
The other string changes were pulled in automatically by the build (from where?) except for this one which I changed to get the helptext updated.  What is the correct way to update this string, @sujato ?

Reply:
I think this has to be updated in `bilara-data` and then go through all its magic.
I'm wondering if this help text should get its own new segment.

Reply:
@thesunshade - Can you take the lead on this upstream change? :pray:

Reply:
Sure. Just give me a poke when the shortcut is live if I don't notice.